keanu-project/src/main/java/io/improbable/keanu/network/ProtobufSaver.java

```diff
@@ -16,28 +16,55 @@
 import java.io.IOException;
 import java.io.OutputStream;
 import java.lang.reflect.Method;
+import java.util.Map;
 import java.util.function.Consumer;
 
 public class ProtobufSaver implements NetworkSaver {
     private final BayesianNetwork net;
-    private KeanuSavedBayesNet.BayesianNetwork.Builder bayesNetBuilder = null;
+    private KeanuSavedBayesNet.Model.Builder modelBuilder = null;
 
     public ProtobufSaver(BayesianNetwork net) {
         this.net = net;
     }
 
+    public KeanuSavedBayesNet.Model getModel(boolean withSavedValues, Map<String, String> metadata) {
```

Comment:
Do we need to make this public?  I'd imagine package private would be better (assuming this is just for the JSON writer).

Reply:
The problem is that we have protobuf in `network` package and json and dot in `utilities`. I was actually about to ask you about it - where do you think we should keep them?

Reply:
They should probably all live in the same location given they all do the same job.  Perhaps if we stick them all in .util (with the non-DotSaver dot related classes in .util.dot ?

Reply:
or put them all in a new package with a suitable name like `io`